ocp_resources/limit_range.py

```diff
@@ -0,0 +1,26 @@
+# API reference: https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-limit-ranges.html
+
+from ocp_resources.resource import NamespacedResource
+
+
+class LimitRange(NamespacedResource):
+    api_version = NamespacedResource.ApiVersion.V1
+
+    def __init__(
+        self,
+        limits=None,
+        **kwargs,
+    ):
+        """
+        Create LimitRange object.
+
+        Args:
+            limits (list): List of limits
+        """
+        super().__init__(**kwargs)
+        self.limits = limits
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file:
+            self.res.setdefault("spec", {})["limits"] = self.limits
```

Comment:
Should limit be required? 
If user create this resource and do not send limit, is this ok?

Reply:
technically, it can be created with empty limits:

```
$ oc get limitrange limitrange1 -o json
{
    "kind": "LimitRange",
    "metadata": {
        "name": "limitrange1",
.
    },
    "spec": {
        "limits": null
    }
}
```

Reply:
ack

Reply:
this looks like a bug, according to the api doc, limits.type is required

Reply:
not sure it is a bug.. as I see there are some tests in kubernetes repository also using it without setting limits:

https://github.com/kubernetes/kubernetes/blob/master/test/integration/auth/rbac_test.go#L249C1-L255C2

Reply:
please update docsting that `limits` is optional

Reply:
done